docs/TEAMS.md

```diff
@@ -0,0 +1,202 @@
+The Organic Maps is led by contributors from across the community. Different functional domains may periodically establish, develop, or dissolve their own leadership structures, whether formal or informal.
+
+The file contains the functional project structure. The structure of GitHub teams must match the content of this file. Any changes to memberships must be performed via a PR to this file.
+
+- [Android Team](#android-team)
+- [iOS Team](#ios-team)
+- [Map Data Team](#map-data-team)
+- [Map Styles Team](#map-styles-team)
+- [DevOps Team](#devops-team)
+- [Design Team](#design-team)
+- [Website Team](#website-team)
+- [Growth Team](#growth-team)
+- [Translation Team](#translation-team)
+- [User Support Team](#user-support-team)
+- [Legal Team](#legal-team)
+
+## Android Team
+
+[@organicmaps/android](https://github.com/orgs/organicmaps/teams/android/members)
+
+**Scope**:
+
+Android application development.
+
+**Members:**
+
+- @AndrewShkrob
+- @biodranik
+- @kavikhalique
+- @map-per
+- @RedAuburn
+- @rtsisyk
+- @vng
+
+## iOS Team
+
+[@organicmaps/ios](https://github.com/orgs/organicmaps/teams/core/members)
+
+**Scope:**
+
+iOS application development.
+
+**Members:** 
+
+- @biodranik
+- @fabwu
+- @kirylkaveryn
+- @vng
+
+## Map Data Team
+
+[@organicmaps/data](https://github.com/orgs/organicmaps/teams/data/members)
+
+**Scope:**
+
+Map data preparation and generation.
```

Comment:
I'm not sure it makes sense to separate Map Data and Map Styles team.

Planet generation for upcoming releases should be a part of Releng / Devops, while the generator changes are usually coupled with map styling, classificator, etc.

@vng WDYT?